.eslintrc.js

```diff
@@ -3,7 +3,7 @@
 module.exports = {
     env: {
         browser: true,
-        es6: true,
+        es2020: true,
```

Comment:
I think we can avoid having to switch to es2020, and keep compatibility with environments without bigint. 

You can add 

```js
if (typeof BigInt !== "function") throw new Error("BigInt not supported");
```

before using it.

Reply:
Will do, thanks